src/forms/CurrentExpenses.js

```diff
@@ -286,12 +297,16 @@ const ExpensesFormContent = function ({ current, time, setClientProperty }) {
             { ...sharedProps }
             generic={ 'childOtherCare' }> Other care 
           </CashFlowRow>
-          <CashFlowRowAfterConfirm
-            { ...sharedProps }
-            generic={ 'earnedBecauseOfChildCare' }
-            confirmLabel={ 'Does childcare allow you to make additional income?' }>
-            <span style={{ textDecoration: 'underline' }}>Income</span> made possible by child care expenses
-          </CashFlowRowAfterConfirm>
+
+          <EarnedFrom
+            heading={ 'Effect of Childcare' }
```

Comment:
Are we treating "childcare" as one or two words? Should be consistent. A quick google search seems to indicate "child care" is more common.